data/json/items/chemicals_and_resources.json

```diff
@@ -132,644 +133,637 @@
+    "volume": 1,
+    "weight": 460,
+    "bashing": 1,
+    "ammo_type": "components",
+    "charges": 2,
+    "phase": "liquid",
+    "container": "bottle_glass",
+    "drop_action": { "type": "emit_actor", "emits": [ "emit_acid_drop" ], "scale_qty": true }
+  },
+  {
+    "type": "AMMO",
+    "id": "chem_acetone",
+    "category": "chems",
+    "price": 50,
+    "name": "acetone",
+    "name_plural": "acetone",
+    "symbol": "=",
+    "color": "white",
+    "description": "Acetone, best known as the primary constituent of most nail polish removers.  Flammable and can be easily used for potent home-made explosives, as demonstrated by the terrorist attacks of 2016.",
+    "volume": 1,
+    "weight": 200,
+    "bashing": 1,
+    "ammo_type": "components",
+    "phase": "liquid",
+    "container": "bottle_glass"
+  },
+  {
+    "type": "AMMO",
+    "id": "chem_nitric_acid",
+    "category": "chems",
+    "price": 300,
+    "name": "nitric acid",
+    "name_plural": "nitric acid",
+    "symbol": "=",
+    "color": "white",
+    "description": "Nitric acid, a strong oxidant and extremely corrosive material.  Mainly used in the production of synthetic fertilizers, for etching of circuit boards and in woodworking.  Remains quite useful both for making a vast array of pyrotechnic and explosive agents and as a direct weapon: not many beings enjoy being doused in nitric acid.",
+    "volume": 1,
+    "weight": 375,
+    "bashing": 1,
+    "ammo_type": "components",
+    "charges": 2,
+    "phase": "liquid",
+    "container": "bottle_glass",
+    "drop_action": { "type": "emit_actor", "emits": [ "emit_acid_drop" ], "scale_qty": true }
+  },
+  {
+    "type": "AMMO",
+    "id": "chem_chromium_oxide",
+    "category": "chems",
+    "price": 500,
+    "name": "chromium oxide",
+    "name_plural": "chromium oxide",
+    "symbol": "=",
+    "color": "green",
+    "description": "A handful of chromium oxide pellets.  This substance, typically known as the only stable green pigment for paints, is also a mild oxidizer, which can be mixed with aluminum powder to make thermite for welding.  If one had access to a CVD machine and a laboratory worth of reagents, they might be able to produce very durable ceramic layers.",
+    "material": "powder",
+    "volume": 1,
+    "weight": 13.05,
+    "bashing": 1,
+    "ammo_type": "components",
+    "container": "bag_canvas",
+    "count": 100
+  },
+  {
+    "type": "AMMO",
+    "id": "chem_carbide",
+    "category": "chems",
+    "price": 75,
+    "name": "calcium carbide",
+    "name_plural": "calcium carbide",
+    "symbol": "=",
+    "color": "white",
+    "description": "A handful of calcium carbide chunks, giving off a faint, garlic-like smell.  Commercially, calcium carbide was used for on-the-spot manufacture of acetylene and historically to power acetylene lamps.  If mixed with water and sealed, it will explode with little strength, but with relatively loud noise and could maybe be used as a decoy...",
```

Comment:
```suggestion
    "description": "A handful of calcium carbide chunks, giving off a faint, garlic-like smell.  Commercially, calcium carbide was used for on-the-spot manufacture of acetylene and historically to power acetylene lamps.  If mixed with water and sealed, it will explode with little strength, but with relatively loud noise and could maybe be used as a decoy…",
```